its/src/test/java/com/sonar/it/csharp/CoverageTest.java

```diff
@@ -137,6 +128,21 @@ public void visual_studio() throws Exception {
     assertLineCoverageMetrics("CoverageTest", 2, 1);
   }
 
+  @Test
+  public void visual_studio_on_MultipleProjects() throws Exception {
+    BuildResult buildResult = analyzeMultipleProjectsTestProject("sonar.cs.vscoveragexml.reportsPaths", "VisualStudio.coveragexml");
+
+    assertThat(buildResult.getLogs()).contains(
+      "Sensor C# Tests Coverage Report Import",
+      "Coverage Report Statistics: 4 files, 3 main files, 3 main files with coverage, 1 test files, 0 project excluded files, 0 other language files.");
+
```

Comment:
It's interesting to see the difference with opencover:

On opencover:

> Coverage Report Statistics: 5 files, 3 main files, 3 main files with coverage, 2 test files, 0 project excluded files, 0 other language files

On VS:

> Coverage Report Statistics: 4 files, 3 main files, 3 main files with coverage, 1 test files, 0 project excluded files, 0 other language files.

I would expect to see 2 test files also for VS (`FirstProjectTests.cs` and `FirstClassTests.cs`)

Reply:
good catch; I noticed this but I only ignored it saying that "meh, vs coverage is weird / stuped" but *I was wrong*.

The problem was that for `SecondProjectTests\FirstClassTests.cs` I missed making the path relative (the manual step)

Reply:
Looks good now, the match is almost perfect 😄 

```
OpenCover:
Coverage Report Statistics: 5 files, 3 main files, 3 main files with coverage, 2 test files, 0 project excluded files, 0 other language files
VS:
Coverage Report Statistics: 5 files, 3 main files, 3 main files with coverage, 2 test files, 0 project excluded files, 0 other language files.
```